README.md

```diff
@@ -4,4 +4,74 @@
 [![Container Repository on Quay](https://quay.io/repository/stackanetes/nova-kubernetes-drain/status "Container Repository on Quay")](https://quay.io/repository/stackanetes/nova-kubernetes-drain)
 [![Go Report Card](https://goreportcard.com/badge/stackanetes/nova-kubernetes-drain "Go Report Card")](https://goreportcard.com/report/stackanetes/nova-kubernetes-drain)
 
+The main goal of Nova-kubernetes-drain is to perform evacuation of [Openstack] compute node when [Kubernetes] node is being drained.
```

Comment:
I'd use `live-evacuation` because the common meaning of `evacuation` in OpenStack world suggests that the node has failed.